src/ansys/mechanical/core/run.py

```diff
@@ -170,6 +171,11 @@ def cli(
     if input_script:
         args.append("-script")
         args.append(input_script)
+        warnings.warn(
```

Comment:
@klmcadams as of last night's build, this was fixed for 2024 R1.  So we should have a version check before this warning

Reply:
@koubaa This is being addressed in #323